common/templates/framework-section.njk

```diff
@@ -49,7 +49,7 @@
             {{ step.pageTitle }}
           </h2>
 
-          {% if canAccess('person_escort_record:update') and isEditable %}
```

Comment:
The rest of this PR is effectively a simple rename - why is the `canAccess` being removed here?

Reply:
Is part of [this commit](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/pull/1102/commits/ec05faf2418613d6fa113ddbffaf25a1061ae49a) which makes this check dynamic based on the assessment type rather than just check for person escort record permissions.

Reply:
You've rebased, so that commit has vanished forever!

But that makes sense :)